.github/tw-rules.yaml

```diff
@@ -0,0 +1,12 @@
+runChecks: true
+actions:
+  branch-protection-settings:
+    branches:
+      - name: main
+        dismissStaleReviews: true
+        requireBranchUpToDate: true
+        checks:
+          - name: Build & Test
+            type: tests
+  sync-code-owners:
+    extraWriters: ["plastic-issuance-service-owners"]
```

Comment:
can you add `full-time-technical-staff` ? https://github.com/orgs/transferwise/teams/full-time-technical-staff
it should have been the default setting but not sure what happened to this repo.

Reply:
Added it now.